pkg/healthcheck/healthcheck.go

```diff
@@ -1353,6 +1370,27 @@ func (hc *HealthChecker) checkCanPerformAction(verb, namespace, group, version,
 	)
 }
 
+func (hc *HealthChecker) checkHAMetadataPresentOnKubeSystemNamespace() error {
+	if hc.isHA() {
+		ns, err := hc.kubeAPI.CoreV1().Namespaces().Get("kube-system", metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		val, ok := ns.Labels[k8s.AdmissionWebhookLabel]
+		if !ok || val != "disabled" {
+			return fmt.Errorf("kube-system namespace needs to have %ss: disabled if HA mode is enabled", k8s.AdmissionWebhookLabel)
+		}
+
+		val, ok = ns.Annotations[k8s.ProxyInjectAnnotation]
+		if !ok || val != "disabled" {
```

Comment:
is this actually required?  the documentation only mentions the `config.linkerd.io/admission-webhooks` label.  https://linkerd.io/2/features/ha/